### PR TITLE
Fixed the issue with mouse clicks moving on high dpi screens, also fi…

### PR DESCRIPTION
**Pull request by danieldeng2**
Fixed the issue with mouse clicks moving on high dpi screens (issue #160), also fixed the issue of not being able to control VLC media player when paused.

### CODE REVIEW

**Comment by flipflop97** [comment]
We've fixed the same thing in exactly the same way apparently.

#177

**Comment by danieldeng2** [comment]
@flipflop97 I saw your original post, didn't know you also fixed it lol

File: src/service/plugins/mpris.js [hunk 1]
```diff
@@ -344,7 +344,7 @@ var Plugin = new Lang.Class({
                 pos: Math.round( pos / 1000),
                 isPlaying: (player.PlaybackStatus === "Playing"),
                 canPause: (player.CanPause === true),
-                canPlay: (player.CanPlay === true),
+                canPlay: (player.CanPlay === true || player.PlaybackStatus === "Paused"),
```
Comment (andyholmes) [inline src/service/plugins/mpris.js]: I think this should be filed as an upstream bug with VLC. `CanPlay` is a [mandatory property](https://specifications.freedesktop.org/mpris-spec/latest/Player_Interface.html#Property:CanPlay) of the MPRIS specification and implementing workarounds for specific players is not a path I want to go down. KDE Connect also uses [MPRIS in the almost the exact same way](https://github.com/KDE/kdeconnect-kde/blob/master/plugins/mpriscontrol/mpriscontrolplugin.cpp#L137-L207) (`CanPlay` means the method `Play()` can be invoked) so I think it's best to stick with that.

**Comment by andyholmes** [comment]
Aside from the note I left, I'm going to merge #177 since it came first (I'll also be leaving a note on that PR)

File: src/service/plugins/mpris.js (hunk 1 above)
Comment (danieldeng2) [reply src/service/plugins/mpris.js]: Ok thanks I respect your decision

**Comment by danieldeng2** [comment]
@andyholmes Also, the seek bar on android defaults back to 0:00 after setting position.

**Pull request closed without merging**